File: dashboard/src/app/model/hospitalMetadataModel.ts
import { IEntityBase }              from  './IEntityBase';
import { MetaType }                 from  './enumModel';
import { WriteHistoryBaseModel }    from './writeHistoryBaseModel';

export class HospitalMetadataModel extends WriteHistoryBaseModel implements IEntityBase{

    id: number;
    hospitalId: number;
    metaType: MetaType;
    jsonValue: string;   

    constructor(){
        super();
        this.id = 0;
        this.hospitalId = 0;
        this.metaType = null;
        this.jsonValue = null;
    } 
}

File: server/medicore.layer/MediCore.Service/Model/HospitalMetadataModel.cs
using dna.core.service.Models.Abstract;
using MediCore.Data.Infrastructure;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MediCore.Service.Model
{
    public class HospitalMetadataModel : WriteHistoryBaseModel, IModelBase
    {

        public int Id { get; set; }
        public int HospitalId { get; set; }
        public MetaType MetaType { get; set; }
        public string JsonValue { get; set; }
    }
}
